### DISCUSSION TITLE
Company-clang and doc popups

### DISCUSSION THREAD

**Discussion by ghost** [company-mode/company-mode#1198] [Q&A] [answered]
Hi there! I am wondering if it is possible to get company-quickhelp to popup documentation tips for c/c++ code using company-clang? I've been searching for awhile now and I can't seem to get it to work, though it seems to be possible?

Any ideas?

**Comment by dgutov** [comment, marked as answer]
Not supported, I'm afraid. There's been an effort to add this feature at https://github.com/company-mode/company-mode/pull/288, but it seems to have stalled.

You might want to try one of the third-party backends: https://github.com/company-mode/company-mode/wiki/Third-Party-Packages#cc
